pymodbus/other_message.py

```diff
@@ -473,24 +485,3 @@ def __str__(self):
             "ReportSlaveIdResponse(%s, %s, %s)"  # pylint: disable=consider-using-f-string
             % arguments
         )
-
-
-# ---------------------------------------------------------------------------#
```

Comment:
@janiversen I removed this to fix the pylint message, I believe this was a stale comment anyways

https://github.com/PyCQA/pylint/issues/1681